index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <div class="toolbar logo">John Doe</div>
+  <nav class="toolbar"><img src="image/Union.png" class="imagcss" alt="dropdown menu"></nav></header>
+  <div class="headline">
```

Comment:
- [ ] As the previous reviewer requested, please make sure to align the hamburger button to the left, it is still appears centered.: 
<img width="368" alt="Screenshot 2022-03-18 at 12 20 07" src="https://user-images.githubusercontent.com/77038610/158963019-a9cde744-e5a9-4bfa-8af6-61690c3a49b9.png">

- [ ] You are using display absolute for nav, I do not recommend using it. You can use` justify-content: space-between` to align the elements:
<img width="381" alt="Screenshot 2022-03-18 at 12 33 46" src="https://user-images.githubusercontent.com/77038610/158965037-44442ba1-a0e3-4df5-afad-3e3ca9f1afec.png">

This is what I changed: 

```
body{
    margin: 0
}

.header {
    width: 375px;
    height: 51px;
    background: #3c3a39;
    display: flex;
    justify-content: space-between;
    align-items: center;
}
.toolbar {
    width: 60px;
    display: flex;
    justify-content: center;
}
```

Reply:
Thanks for code I add them but it required more changes but am fixing it